src/java.base/share/classes/java/util/Properties.java

```diff
@@ -165,6 +168,10 @@
      */
     private transient volatile ConcurrentHashMap<Object, Object> map;
 
+    // used to format the date comment written out by the store() APIs.
+    // This format matches the one used by java.util.Date.toString()
+    private static final String dateFormatPattern = "EEE MMM dd HH:mm:ss zzz yyyy";
```

Comment:
I'd rename this something like DATE_FORMAT_PATTERN to conform to naming conventions for constants.